src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -38,17 +42,25 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'choice_loader' => function (Options $options) {
                 $regions = $options->offsetGet('regions', false);
+                $input = $options->offsetGet('input');
```

Comment:
We should use `->offsetGet()` method when we need to pass the second argument, otherwise use `$options['input']` for consistence only.